mongoDB/sampleData.js

```diff
@@ -0,0 +1,903 @@
+module.exports.pictures = [{ "item_id": 1, "store_id": 8, "item_pictures": [{ "large": "https://picsum.photos/750/1000", "normal": "https://picsum.photos/340/270", "thumbnail": "https://picsum.photos/75/75" }, { "large": "https://picsum.photos/750/1000", "normal": "https://picsum.photos/340/270", "thumbnail": "https://picsum.photos/75/75" }, { "large": "https://picsum.photos/750/1000", "normal": "https://picsum.photos/340/270", "thumbnail": "https://picsum.photos/75/75" }, { "large": "https://picsum.photos/750/1000", "normal": "https://picsum.photos/340/270", "thumbnail": "https://picsum.photos/75/75" }, { "large": "https://picsum.photos/750/1000", "normal": "https://picsum.photos/340/270", "thumbnail": "https://picsum.photos/75/75" }], "seller_picture": "https://picsum.photos/75/75", "store_picture": "https://picsum.photos/140/140" },
+{ "item_id": 2, "store_id": 1, "item_pictures": [{ "large": "https://picsum.photos/750/1000", "normal": "https://picsum.photos/340/270", "thumbnail": "https://picsum.photos/75/75" }, { "large": "https://picsum.photos/750/1000", "normal": "https://picsum.photos/340/270", "thumbnail": "https://picsum.photos/75/75" }, { "large": "https://picsum.photos/750/1000", "normal": "https://picsum.photos/340/270", "thumbnail": "https://picsum.photos/75/75" }, { "large": "https://picsum.photos/750/1000", "normal": "https://picsum.photos/340/270", "thumbnail": "https://picsum.photos/75/75" }, { "large": "https://picsum.photos/750/1000", "normal": "https://picsum.photos/340/270", "thumbnail": "https://picsum.photos/75/75" }, { "large": "https://picsum.photos/750/1000", "normal": "https://picsum.photos/340/270", "thumbnail": "https://picsum.photos/75/75" }], "seller_picture": "https://picsum.photos/75/75", "store_picture": "https://picsum.photos/140/140" },
```

Comment:
Looks like these are all the same url - if you plan on using all the same url and that url generates a random picture every time it seems like this could be simplified somehow. Do you foresee any problems with re-using the same url?